metis-harvesting/src/main/java/eu/europeana/metis/harvesting/http/HttpHarvester.java

```diff
@@ -79,29 +50,7 @@ HttpRecordIterator createTemporaryHttpHarvestIterator(InputStream input,
    * @return A HttpRecordIterator based on a temporary file location
    * @throws HarvesterException In case there is an issue while using the input stream
    */
-  FullRecordHarvestingIterator<ArchiveEntry, Path> createFullRecordHarvestIterator(InputStream input,
+  FullRecordHarvestingIterator<FullRecord, Path> createFullRecordHarvestIterator(InputStream input,
       CompressedFileExtension compressedFileType) throws HarvesterException;
 
-  /**
-   * An object representing an entry in a file archive. The harvesting identifier is the file name
-   * (including the path relative to the archive root).
-   */
-  interface ArchiveEntry extends FullRecord {
```

Comment:
this will impact the sandbox.

Reply:
You are right! I will create a PR there as well then.